example/lib/main.dart

```diff
@@ -95,150 +95,152 @@ class _MyAppState extends State<MyApp> {
+                    );
+                  },
+                ),
+                SizedBox(height: 16),
+                Text(
+                  "BG Processing Tasks (iOS only)",
+                  style: Theme.of(context).textTheme.headline5,
+                ),
+                //This task runs once.
+                //Most likely this will trigger immediately
+                PlatformEnabledButton(
+                  platform: _Platform.ios,
+                  child: Text("Perform a BG Task"),
+                  onPressed: () {
+                    Workmanager().registerOneOffTask(
+                      "vn.newwave.custom-task-identifier",
+                      simpleTaskKey,
+                      inputData: <String, dynamic>{
+                        'int': 1,
+                        'bool': true,
+                        'double': 1.0,
+                        'string': 'string',
+                        'array': [1, 2, 3],
+                      },
+                    );
+                  },
+                ),
 
-              Text(
-                "One Off Tasks (Android only)",
-                style: Theme.of(context).textTheme.headline5,
-              ),
-              //This task runs once.
-              //Most likely this will trigger immediately
-              PlatformEnabledButton(
-                platform: _Platform.android,
-                child: Text("Register OneOff Task"),
-                onPressed: () {
-                  Workmanager().registerOneOffTask(
-                    "1",
-                    simpleTaskKey,
-                    inputData: <String, dynamic>{
-                      'int': 1,
-                      'bool': true,
-                      'double': 1.0,
-                      'string': 'string',
-                      'array': [1, 2, 3],
-                    },
-                  );
-                },
-              ),
-              PlatformEnabledButton(
-                platform: _Platform.android,
-                child: Text("Register rescheduled Task"),
-                onPressed: () {
-                  Workmanager().registerOneOffTask(
-                    "1-rescheduled",
-                    rescheduledTaskKey,
-                    inputData: <String, dynamic>{
-                      'key': Random().nextInt(64000),
-                    },
-                  );
-                },
-              ),
-              PlatformEnabledButton(
-                platform: _Platform.android,
-                child: Text("Register failed Task"),
-                onPressed: () {
-                  Workmanager().registerOneOffTask(
-                    "1-failed",
-                    failedTaskKey,
-                  );
-                },
-              ),
-              //This task runs once
-              //This wait at least 10 seconds before running
-              PlatformEnabledButton(
+                Text(
+                  "One Off Tasks (Android only)",
```

Comment:
Is that still the case? (Android only)

Reply:
You're right. We don' need any the specific button any more.